How to configure the target path directory that artifacts are written to (#4921)
fishtown-analytics/docs.getdbt.com: website/docs/reference/artifacts/dbt-artifacts.md

## When are artifacts produced?

Most dbt commands (and corresponding RPC methods) produce artifacts:
- [semantic manifest](/docs/dbt-cloud-apis/sl-manifest): produced whenever your dbt project is parsed
- [manifest](/reference/artifacts/manifest-json): produced by commands that read and understand your project
- [run results](/reference/artifacts/run-results-json): produced by commands that run, compile, or catalog nodes in your DAG
- [catalog](catalog-json): produced by `docs generate`
- [sources](/reference/artifacts/sources-json): produced by `source freshness`

## Where are artifacts produced?

> By default, artifacts are written to the `/target` directory of your dbt project, and this location is configurable via the [`target-path`](/reference/project-configs/target-path#configuration).

Reviewer: ```suggestion
By default, artifacts are written to the `/target` directory of your dbt project. You can configure the location using the [`target-path`](/reference/project-configs/target-path#configuration).
```
